src/lib/libast/features/sig.sh

```diff
@@ -105,4 +105,5 @@ extern int		sigflag(int, int, int);
 extern int		sigflag(int, int, int);
 #endif
 extern int		sigcritical(int);
-extern int		sigunblock(int);'
+extern int		sigunblock(int);
+extern Sig_handler_t	_ast_signal(int, Sig_handler_t);'
```

Comment:
This still has `_ast_signal`. Probably not needed now (?)

Reply:
It is still required to avoid implicit function warnings on Haiku (due to the unique situation with `signal` in the OS headers).